### DISCUSSION TITLE
Can the Mel spectrogram inverters be trained standalone?

### DISCUSSION THREAD

**Discussion by AI-Guru** [coqui-ai/TTS#2042] [General]
Hi!

Thanks a lot for the great work you are doing! Fantastic that you do a lot of open source work!

I am very curious. Can the Mel spectrogram inverters be trained standalone? You list a lot of Vocoders (MelGAN, Multiband-MelGAN, GAN-TTS, ParallelWaveGAN, WaveGrad, WaveRNN) in your README. Is there some example how to train those standalone?

Best,
Tristan

**Comment by erogol** [comment]
Thanks for your comments :) Yes you can train them. See https://tts.readthedocs.io/en/latest/

**Reply by AI-Guru** [reply]
Ah! Thanks! I missed that!

https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html#training-a-vocoder-model
